src/styles/_catalog.scss

```diff
@@ -1,9 +1,33 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$card-width: 200px;
+$card-gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+.cards-block {
+  display: grid;
+  justify-content: center;
+
+  &__all-cards {
+    display: grid;
+    justify-content: space-between;
+    gap: $card-gap;
+  }
+
+  @media (min-width: 488px) {
+    &__all-cards {
+      grid-template-columns: repeat(2, minmax(200px, 1fr));
+    }
+  }
+
+  @media (min-width: 768px) {
+    &__all-cards {
+      grid-template-columns: repeat(3, minmax(200px, 1fr));
+    }
+  }
+
+  @media (min-width: 1024px) {
+    &__all-cards {
+      grid-template-columns: repeat(4, minmax(200px, 1fr));
+    }
+  }
```

Comment:
move all `@media` inside `all-cards` selector and get rid of selectors inside media

Reply:
also you have card-width variable, but didn't used it